examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py

```diff
@@ -460,7 +460,7 @@ def main():
     # if SpecAugment is used for whisper models, return attention_mask to guide the mask along time axis
     forward_attention_mask = (
         getattr(config, "model_type", None) == "whisper"
-        and getattr(config, "apply_spec_augment", False)
+        and getattr(config, "apply_spec_augment", True)
```

Comment:
Hey @sorgfresser - this is just checking whether the model config has the attribute `apply_spec_augment`:
* If the config has the attribute `apply_spec_augment`, then we return the value in the config
* If it does **not** have the attribute `apply_spec_augment`, then we return `False`

The original Whisper models were **not** trained with Spec Aug, thus if they do not have the attribute `apply_spec_aug` in the config, the code defaults to not using Spec Aug (`False`). So the code is correct in its current state

If you want to train the model with Spec Aug enabled, pass the argument `--apply_spec_augment` when you launch training, and the config attribute `apply_spec_aug` will be set to `True`.

=> this will enable Spec Aug in the fine-tuning run